src/operator/numpy/np_percentile_op-inl.h

```diff
@@ -65,6 +66,17 @@ struct NumpyPercentileParam : public dmlc::Parameter<NumpyPercentileParam> {
     DMLC_DECLARE_FIELD(q_scalar).set_default(dmlc::optional<double>())
       .describe("inqut q is a scalar");
   }
+  void SetAttrDict(std::unordered_map<std::string, std::string>* dict) {
+    std::ostringstream axis_s, interpolation_s, keepdims_s, q_scalar_s;
+    axis_s << axis;
+    interpolation_s << interpolation;
+    keepdims_s << keepdims;
+    q_scalar_s << q_scalar;
+    (*dict)["axis"] = axis_s.str();
+    (*dict)["interpolation"] = interpolation_s.str();
```

Comment:
Similar to `String2MXNetPercentileType`, we have to `MXNetPercentileType2String` here. It can be added in `incubator-mxnet/src/api/operator/op_utils.cc`.